app/models/driver_commission_history_adapter.rb

```diff
@@ -0,0 +1,52 @@
+# DriverCommissionHistoryAdapter
+class DriverCommissionHistoryAdapter
+  # self.using(:grossman).connection.current_shard
+  QUERY_ALL = <<SQL
+        SELECT
+          DriverId
+        , Backhauls
+        , FreightRevenue
+        , CustomerId
+        , DeliveryDate
+        , DriverRate
+        , FreightBillNumber
+        , FuelSurcharge
+        , Layover
+        , MovementType
+        , OtherHourDollars
+        , OrderNumberText
+        , OriginName
+        , PaidDate
+        , Revenue
+        , SplitRate
+        , TotalFreightRevenue
+        FROM
+          Trucking_Drivers_Commissions_History
+SQL
+
+  QUERY_BY_PAID_DATE = QUERY_ALL + 'WHERE PaidDate = ?'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.